lib/remote/httpserverconnection.cpp

```diff
@@ -348,8 +348,6 @@ bool EnsureValidBody(
 		Array::Ptr permissions = authenticatedUser->GetPermissions();
 
 		if (permissions) {
-			CpuBoundWork evalPermissions (yc);
-
```

Comment:
## Optional

A perfect example of a separate PR which is just ways faster to review. :-)

Reply:
> which is just ways faster to review. :-)

This is a one-line change in a separate commit which does not make reviewing any faster nor slower.